src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
+
+## Eligibility
+
+Avoid using the start point to provide complex eligibility information — instead, ask questions inside the service to establish whether the user is eligible.
+
+Read [guidance on designing how GOV.UK content and transactions work together](https://www.gov.uk/service-manual/design/govuk-content-transactions).
+
+## Choosing a format for a start point on GOV.UK
+
+The options available depend on whether the start point will be published as ‘mainstream’ or ‘Whitehall’ content.
+
+The content team at GDS is responsible for creating mainstream start points (as with other mainstream content). ‘Mainstream’ content is content aimed at the general public — or at meeting business needs that aren’t sector-specific.
+
+### Mainstream start points
+
+For a mainstream start point, the options are:
+- a simple start page
+- a start point within a multipart guide
+
+Use a simple start page if users can start using the service without needing much context.
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page, taken from the coronavirus shielding support service. The page includes a heading, a few simple paragraphs and bullet points to explain who can use the service, and a start button labelled 'Register or update your details'.">
+
+If the start point is linked to a wider process that needs more explanation, create an ‘Apply’ section within a multipart guide. 
 
-Start pages include 4 main elements:
+For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, how to get the relevant information together and how the different outcomes affect what they'll need to do next.
 
-1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+It’s important that users have this context, so it makes sense to embed the start point within the multipart guide — rather than creating the start point as a separate start page, allowing users to miss the context and potentially do the wrong thing.
 
-2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+<img src="multipart-guide-start-point.png" alt="A screenshot showing an example of a start point within a multipart guide. The page is shown as page 4 of 5 of a guide and its body includes a heading, a few simple paragraphs and bullet points to explain what the user will need to start using the service, and a start button labelled 'Apply now'.">
```

Comment:
Alt text is 297 chars (think GOV UK limit is 240).

Reply:
```suggestion
<img src="multipart-guide-start-point.png" alt="A screenshot showing an example of a start point within a multipart guide. The page is shown as page 4 of 5 and its body text explains what the user will need to start using the service, followed by a start button labelled 'Apply now'.">
```